src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -184,43 +181,43 @@
     <value>An operating system invoke method is defined and a method that has the equivalent functionality is located in the .NET Framework class library.</value>
   </data>
   <data name="UseManagedEquivalentsOfWin32ApiMessage" xml:space="preserve">
-    <value>Use managed equivalents of win32 api</value>
+    <value>Use managed equivalents of win32 api.</value>
```

Comment:
I thought we decided not to have a period for the title and message strings, only for description.

Reply:
I thought we said '.' for messages too but you are right: https://github.com/dotnet/roslyn-analyzers/pull/3183#issuecomment-576724513 I'll do a new pass through the resources then xD